### PR TITLE
Add method to piwik.js that let's you debug which content blocks are on your page

### LINKED ISSUE #8650: Add method to piwik.js that let's you debug which content blocks are on your page
While helping a user to figure out via no content impressions are shown in the UI ( http://forum.piwik.org/read.php?15,122599,129060 ) I noticed it would be very helpful to have a method that let's you get all detected content blocks. Either just the dom nodes or a JSON array containing all content blocks like   

```
[{name: '...', target:'...', piece: "..."},...]
```

It could by default just log to the console via `logAllContentImpressions()`

### PR DESCRIPTION
**Pull request by tsteur**
fixes #8650 

Added new method `logAllContenBlocksOnPage()`.

After merging I will document in JS Tracker: http://developer.piwik.org/api-reference/tracking-javascript and content tracking guide: http://developer.piwik.org/guides/content-tracking

### CODE REVIEW

File: js/piwik.js [hunk 1]
```diff
@@ -5319,6 +5320,20 @@ if (typeof Piwik !== 'object') {
                 },
 
                 /**
+                 * Useful to debug content tracking. This method will log all detected content blocks to console
+                 * (if the browser supports the console). It will list the detected name, piece, and target of each
+                 * content block.
+                 */
+                logAllContenBlocksOnPage: function () {
```
Comment (diosmosis) [inline js/piwik.js]: typo: should be `logAllContentBlocksOnPage`

Comment (diosmosis) [inline js/piwik.js]: Thought about fixing this myself, but since the ticket description uses the apparent misspelling, I will wait for your input.

Comment (tsteur) [inline js/piwik.js]: That's a typo indeed. Thx! Will fix

Comment (tsteur) [inline js/piwik.js]: Done

**Pull request merged**